instances/UBERONParcellation/ansiformLobuleCrusII.jsonld

```diff
@@ -0,0 +1,34 @@
+{
+    "@context": {
+        "@vocab": "https://openminds.ebrains.eu/vocab/"
+    },
+    "@id": "https://openminds.ebrains.eu/instances/UBERONParcellation/ansiformLobuleCrusII",
+    "@type": "https://openminds.ebrains.eu/controlledTerms/UBERONParcellation",
+    "definition": "'Ansiform lobule crus II' is an ansiform lobule and hemisphere part of cerebellar posterior lobe.",
+    "description": "",
+    "interlexIdentifier": "http://uri.interlex.org/base/ilx_0104942",
+    "knowledgeSpaceLink": "https://knowledge-space.org/wiki/NLXANAT:20081209#ansiform-lobule-crus-ii",
+    "name": "ansiform lobule crus II",
+    "preferredOntologyIdentifier": "http://purl.obolibrary.org/obo/UBERON_0005977",
+    "synonym": [

```

Comment:
please only use the exact synonyms of the UBERON term. 
please remove unneeded capitalization (no capitalization except for proper names and abbreviations)

Reply:
I removed one term that was not specific to crus II. All the others are directly from the [UBERON page](http://purl.obolibrary.org/obo/UBERON_0005977). Capitalizations are also removed. I copied these from the crus I file though, so then I guess it is wrong there, too - should I correct that file as well?

Reply:
@eapapp I just realized now: if the crus I also state more than the exact synonyms of the UBERON ontology we should correct the file as well, yes. Unless we definitely know that other terms are also exact synonyms.

Reply:
I would suggest to keep all synonyms in the list, here is why.

Apart from the 11 exact synonyms, the following terms are listed in **ansiformLobuleCrusII.jsonld**.

These terms are listed on the UBERON page under "has related synonym":
 - ansiform lobe of the cerebellum, crus 2
 - Cb-Crus II
 - crus 2
 - crus II
 - lobulus semilunaris caudalis
... and the related term "crus II (cerebelli)", which is not directly listed, but is essentially a shortened form of _crus secundum lobuli ansiformis cerebelli_, listed as an exact synonym.

And one term from NeuroNames, listed on the UBERON page above the synonym list, marked as "latin term": 
 - lobulus posterior inferior
This is a direct equivalent of the English term _posterior inferior lobule_, listed under the exact synonyms.

Reply:
@eapapp we did not take the related synonyms on purpose so far (because they are not exact, meaning ambiguous). We can reassess this again but as long as we do not distinguish schema-wise between different synonym types please just keep the exact ones.

Reply:
I removed the synonyms that are not on the exact list for both **ansiformLobuleCrusI.jsonld** and **ansiformLobuleCrusII.jsonld**. Defs are also deleted from both files.